rebound-android-playground/src/main/java/com/facebook/rebound/playground/examples/BallExample.java

```diff
@@ -70,75 +80,48 @@ public BallExample(Context context, AttributeSet attrs) {
 
   public BallExample(Context context, AttributeSet attrs, int defStyle) {
     super(context, attrs, defStyle);
-    COASTING = SpringConfig.fromOrigamiTensionAndFriction(0, 0.5);
-    COASTING.tension = 0;
     setBackgroundColor(Color.WHITE);
+    configCoasing();
+    configCirclePaint();
+    configBallPaint();
+    configTextPaint();
+  }
 
-    springSystem = SpringSystem.create();
-    springSystem.addListener(this);
-    xSpring = springSystem.createSpring();
-    ySpring = springSystem.createSpring();
-    xSpring.addListener(this);
-    ySpring.addListener(this);
-    paint = new Paint(Paint.ANTI_ALIAS_FLAG);
-
-    getViewTreeObserver().addOnGlobalLayoutListener(new ViewTreeObserver.OnGlobalLayoutListener() {
-      @Override
-      public void onGlobalLayout() {
-        centerX = getWidth() / 2f;
-        centerY = getHeight() / 2f;
-
-        xSpring.setCurrentValue(centerX).setAtRest();
-        ySpring.setCurrentValue(centerY).setAtRest();
-        getViewTreeObserver().removeOnGlobalLayoutListener(this);
-
-        int offsetH = (int) ((getHeight() - (2 * radius)) % 800) / 2;
-        int offsetW = (int) ((getWidth() - (2 * radius))  % 800) / 2;
-        for (float i = offsetH + radius; i < getHeight() - offsetH; i += 400) {
-          for (float j = offsetW + radius; j < getWidth() - offsetW; j += 400) {
-            points.add(new PointF(j, i));
-          }
-        }
+  @Override protected void onSizeChanged(int width, int height, int oldWidth, int oldHeight) {
+    super.onSizeChanged(width, height, oldWidth, oldHeight);
+    centerX = getWidth() / 2f;
+    centerY = getHeight() / 2f;
+    xSpring.setCurrentValue(centerX).setAtRest();
+    ySpring.setCurrentValue(centerY).setAtRest();
+    int offsetH = (int) ((getHeight() - (2 * radius)) % 800) / 2;
+    int offsetW = (int) ((getWidth() - (2 * radius))  % 800) / 2;
+    for (float i = offsetH + radius; i < getHeight() - offsetH; i += 400) {
```

Comment:
Can be, it's from the code of the master branch (by facebook). I don't have done any change there.